src/Connections/GitlabRepo.ts

```diff
@@ -158,6 +177,48 @@ export class GitLabRepoConnection extends CommandConnection {
         }
     }
 
+    public static getProvisionerDetails(botUserId: string) {
+        return {
+            service: "gitlab",
+            eventType: GitLabRepoConnection.CanonicalEventType,
+            type: "GitLabRepo",
+            // TODO: Add ability to configure the bot per connnection type.
+            botUserId: botUserId,
```

Comment:
```suggestion
            botUserId,
```

Also, unsolved TODO in the line above.

Reply:
I'll remove the TODO, it's basically a feature request and I think having these lines makes it more confusing.